.github/workflows/tests.yml

```diff
@@ -237,19 +237,32 @@ jobs:
         if: always()
         run: |
           touch bottles/steps_output.txt
-          cat bottles/steps_output.txt
+
+          echo "# Build failure summary on ${{ matrix.runner }}" >> "$GITHUB_STEP_SUMMARY"
+          echo '```' >> "$GITHUB_STEP_SUMMARY"
+          cat bottles/steps_output.txt | tee -a "$GITHUB_STEP_SUMMARY"
+          echo '```' >> "$GITHUB_STEP_SUMMARY"
+
```

Comment:
Is it worth making these changes part of https://github.com/Homebrew/actions/pull/289?

Reply:
Let's not change the scope of a GSoC project based on maintainer wishes.

Reply:
Sure, but heads up that parts of this might be removed when these steps are replaced with the composite action.